pkg/fakerp/server.go

```diff
@@ -57,70 +63,25 @@ func NewServer(log *logrus.Entry, resourceGroup, address string) *Server {
 	return s
 }
 
-func (s *Server) ListenAndServe() {
-	// TODO: match the request path the real RP would use
-	http.HandleFunc("/", s.handleFunc)
+func (s *Server) Run() {
+	s.router.Use(middleware.DefaultCompress)
+	s.router.Use(middleware.RedirectSlashes)
+	s.router.Use(middleware.Recoverer)
+	s.router.Use(s.logger)
+	s.router.Use(s.validator)
+
+	s.router.Get("/", s.handleGet)
```

Comment:
Open a follow-up issue to update the Location headers we return from deletes so we can stop serving at /.

Reply:
https://github.com/openshift/openshift-azure/issues/1028